docs/src/templates/includes/components/core/menu/je-menu-header.hbs

```diff
@@ -0,0 +1,21 @@
+<header class="c-card c-card--outline c-card--outline--aboveMid c-card--rounded--large c-menuHeader c-mediaElement c-mediaElement--fullstack">
+    <img
+        class="c-mediaElement-img c-mediaElement-img--outlined c-mediaElement-img--negativeTop"
+        src="{{baseUrl}}/assets/img/examples/restaurant-logo.gif"
+        alt="Fozzie Kebab"
+        itemprop="image" />
+    <div class="c-mediaElement-content">
+        <h1 class="text-center u-noSpacing">Fozzie Kebab</h1>
+        <p class="c-badge c-badge--light c-badge--large text-center u-noSpacing">Kebab <span>•</span> Pizza</p>
```

Comment:
What's the •?

Reply:
A separator — https://app.zeplin.io/project/5a7af4078b4a016801348b6c/screen/5b72ec2f9d86cc7bc4ad9511